tests/test_response.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import json
+from test_plus import APITestCase
+
+from wazimap_ng.utils import error_handler
+
+from django.test import RequestFactory
+
+
+@pytest.mark.django_db
+class TestErrorResponseData(APITestCase):
+
+    def test_error_handler(self):
+        factory = RequestFactory()
+        request = factory.get('/api/v1/jkhj')
+
+        response = error_handler(request)
```

Comment:
you're headed in the wrong direction here. 
you want to test the result of the exception raised by DRF and django and not call your own `error_handler`